src/block/stdio_executor.rs

```diff
@@ -117,6 +137,21 @@ impl Display for Error {
     }
 }
 
+/// Errors encountered while processing a request execution result.
+#[derive(Debug)]
+pub enum ProcessReqError {
```

Comment:
Can we use the `Error` type for `process_request` errors instead of defining a new one? The variants seem to be already present (and we could have added new ones if they weren't).

Reply:
Hmmm, that would mean we would have to associate a status with that variant and I think this makes sense only for errors thrown during request execution (not, for example, for an error that is thrown if we fail at writing the status to memory). I would keep a separate enum here, even though it happens that the variants are identical to some of `Error`. Does it make sense?